Source/WebKit/Scripts/webkit/messages.py

```diff
@@ -1821,6 +1933,46 @@ def collect_message_statements(messages, message_statement_function):
     return ''.join(result)
 
 
+def generate_swift_message_handler(receiver):
```

Comment:
we should really, really, really, really use Swift Macros for this, it would give such a better experience and functionality ( know that’s not practical in the short term, but just putting it out there!)